### ISSUE TITLE
Break window is not immediatelly fullscreen

### ISSUE THREAD

**Issue by di72nn** [slgobinath/SafeEyes#365]
Quite often (not always) the break window appears for a fraction of a second as a small non-fullscreen window in the upper left corner of the screen (before eventually switching to fullscreen).

I use i3wm, the problem is reproducible both with and without a compositor.

I experimented a bit and it seems adding `window.resize(monitor_gemoetry.width, monitor_gemoetry.height)` somewhere before `window.present()` fixes that behavior (I currently have `resize` right after the first `window.move(x, y)`).

**Comment by slgobinath** [comment]
Hi, Thank you for reporting this issue.
Do you also have `window.resize(monitor_gemoetry.width, monitor_gemoetry.height)` after `present`?

**Comment by di72nn** [comment]
Yeah, I patched 2.0.9 to have `resize` twice:
```python
            # In Unity, move the window before present
            window.move(x, y)
            window.resize(monitor_gemoetry.width, monitor_gemoetry.height)
            window.stick()
            window.set_keep_above(True)
            window.present()
            # In other desktop environments, move the window after present
            # if self.context['desktop'] != 'unity':
            window.move(x, y)
            window.resize(monitor_gemoetry.width, monitor_gemoetry.height)
            logging.info("Moved break screen to Display[%d, %d]", x, y)
            window.fullscreen()
```

**Comment by slgobinath** [comment]
Thank you for the details. Fixed with https://github.com/slgobinath/SafeEyes/commit/40dc69883fe743a180f625d6fce5142ea421a330
